app/src/main/res/values-night/strings.xml

```diff
@@ -0,0 +1,44 @@
+<resources>
```

Comment:
Why are you creating extra strings resources for night mode

Reply:
yeah ! i just needed the styles.xml

Reply:
I have never squashed commits before but since I have made 3 commits ,would this work

git rebase -i HEAD~3

Reply:
yeah do this and when the new dialog box appears then remove pick in upper 2 and put r and then for last pick put f and then save it

Reply:
I did that and stil i am unable to push
![Kazam_screenshot_00043](https://user-images.githubusercontent.com/42780760/102577841-d5fec500-411e-11eb-93c5-928404a43cb8.png)